wdl2cwl/tests/cwl_files/validateOptimus_5.cwl

```diff
@@ -51,7 +51,7 @@ requirements:
   - class: ResourceRequirement
     coresMin: 1
     ramMin: 953.67431640625
-    outdirMin: 1024
+    outdirMin: $((1) * 1024)
```

Comment:
If there are only static values, can we calculate the result instead of using a CWL expression?